src/core/meshcop/border_agent.cpp

```diff
@@ -280,6 +280,14 @@ BorderAgent::CoapDtlsSession *BorderAgent::FindActiveCommissionerSession(void)
         }
     }
 
+#if OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
+    CoapDtlsSession *coapSession = GetEphemeralKeyManager().mCoapDtlsSession;
```

Comment:
Thanks for finding this and the PR. The change is good, but the style needs to be updated. We cannot declare a variable in the middle of the code. All local variables should be at the top of the scope (per the style guide).

We don't actually need a local variable here; we can change it to this:

```c++
#if OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
    if ((mEphemeralKeyManager.mCoapDtlsSession != nullptr) &&
        mEphemeralKeyManager.mCoapDtlsSession->IsActiveCommissioner())
    {
        commissionerSession = mEphemeralKeyManager.mCoapDtlsSession;
    }
#endif
```